### ISSUE TITLE
unable to parse config file, regenerating

### ISSUE THREAD

**Issue by overrnet** [railwayapp/cli#385]
happens when attempting to use the v3 cli with a config that was generated with the v2 cli

**Comment by malipetek** [comment]
This happened when I renamed a project. Just delete the config file.
```bash
rm -f ~/.railway/config.js
```

**Comment by overrnet** [comment]
yep that's the fix but this issue is to hopefully prevent the config file from corrupting in the first place

**Comment by noam-honig** [comment]
This also happens for me - If I'm trying to unlink a project and init a new one in the same folder - I get this error.

It is easily reproducable
